doc/api/wallet_owner_api.md

```diff
@@ -408,7 +409,7 @@ Send a transaction either directly by http or file (then display the slate)
 ### POST Finalize Tx
 
 Sender finalization of the transaction. Takes the slate returned by the sender as well as the private file generate on the first send step.
-Builds the complete transaction and sends it to a grin node for propagation.
+Builds the complete transaction.
```

Comment:
Maybe change it to: Builds the complete transaction but will **not** send it to the node for propagation. Since there seems to be a lot of confusion